content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_data.md

```diff
@@ -230,6 +235,3774 @@ An example of a successful response:
+            "LowLimit" : {
+               "currency" : "BTC",
+               "issuer" : "rchGBxcD1A1C2tdxF6papQYZ8kjRKMYcL",
+               "value" : "0"
+            },
+            "LowNode" : "0000000000000BD2",
+            "PreviousTxnID" : "8A292FEDE82F23A60E1BC7188F1C88147DAD7303D297A39875DFC8A7DD74C772",
+            "PreviousTxnLgrSeq" : 35731651,
+            "index" : "0004D41DDBE051FD5302886A10C9990F4264DC657CE49F62F95DF3A86E255C2D"
+         },
+         {
+            "Flags" : 0,
+            "Indexes" : [ "FD6EBE613BED36E97D7FCAEF9870FFDEC49FCC23FA76FA7443AB59F66C4DFBF8" ],
+            "LedgerEntryType" : "DirectoryNode",
+            "Owner" : "rJTjmpFeVYxKzDXcxPYyxqDFPsAjHU1szc",
+            "RootIndex" : "0004E7DEEA81686BE43530355C41B0931EB4DB21726E6FDF913CF1425AF30400",
+            "index" : "0004E7DEEA81686BE43530355C41B0931EB4DB21726E6FDF913CF1425AF30400"
+         },
+         {
+            "Balance" : {
+               "currency" : "MTL",
+               "issuer" : "rrrrrrrrrrrrrrrrrrrrBZbvji",
+               "value" : "-9999999999999999e-4"
+            },
+            "Flags" : 2228224,
+            "HighLimit" : {
+               "currency" : "MTL",
+               "issuer" : "rUZt6cD3ZN6iWat1q3Z6B4YmkPBEa88vYE",
+               "value" : "1000000000000000e-3"
+            },
+            "HighNode" : "0000000000000000",
+            "LedgerEntryType" : "RippleState",
+            "LowLimit" : {
+               "currency" : "MTL",
+               "issuer" : "rfe8yiZUymRPx35BEwGjhfkaLmgNsTytxT",
+               "value" : "0"
+            },
+            "LowNode" : "0000000000000000",
+            "PreviousTxnID" : "102CCCA09AC9D0337F1E4E8EE66D7ADC6BCE3E54F1D3785A06C383555E81D08A",
+            "PreviousTxnLgrSeq" : 7117664,
+            "index" : "0004ED7218296FBABFC9790AFE8A193CB8B4E52CCF6D06D2F5CD81A9125BC7D1"
+         },
+         {
+            "Account" : "rhafmcHgwPDhUtwHKJHeS1AowVQ6yrBzry",
+            "Balance" : "186663000",
+            "Flags" : 0,
+            "LedgerEntryType" : "AccountRoot",
+            "OwnerCount" : 0,
+            "PreviousTxnID" : "0B3EF2DEF14E17DEF61DC6489800FCA372A7D2FBBF1344AF690A231EDD722390",
+            "PreviousTxnLgrSeq" : 35410824,
+            "Sequence" : 1,
+            "index" : "0004EFFADB2A92621B66644809830ADC7F5453D96F523D6885D5D4F5953A674C"
+         },
+         {
+            "Account" : "rse7zNjdyUTcuRyAbnmY7GKV2cSSM9xYbx",
+            "Balance" : "20150000",
+            "Flags" : 0,
+            "LedgerEntryType" : "AccountRoot",
+            "OwnerCount" : 0,
+            "PreviousTxnID" : "FF39483D49A3132227A5F83316CCFD943248CA3D5D68161B4177791F9ED384F5",
+            "PreviousTxnLgrSeq" : 30401374,
+            "Sequence" : 4,
+            "index" : "0004F80C0082589154F40E9962406ADF2A6BC943BCD073863B8BB9B3EDB306EA"
+         }
+      ],
+      "status" : "success",
+      "validated" : false
+   }
+}
```

Comment:
Please don't past entire 4000-line code samples into the documentation files. Notice how the other examples use `limit: 5` to reduce the size of the response to something that's actually kind of manageable?

Though, in this case, there isn't commandline syntax for this command anyway so we can skip it.